OONIProbeUnitTests/APITest.m

```diff
@@ -0,0 +1,49 @@
+#import <XCTest/XCTest.h>
```

Comment:
This is just a comment on the directory name. We cannot call this test a "unit test" because it's clearly using the network and relying on its availability. Why are tests folders named like this? Do we want to have a clear separation between unit, integration, and UI tests? Then, probably, we need a folder for integration tests.

Reply:
The dir name is the target name, to change this we should create a new target and this increase complexity. Maybe these tests didn't belong in this PR

Reply:
> Maybe these tests didn't belong in this PR

Tests are always welcome.

You didn't reply to my question about why test names have been chosen like this. What was the rationale?

Reply:
Because it's not a folder, it's a target/scheme:
Arturo choose the second
Xcode choose the third
<img width="210" alt="Screenshot 2019-06-27 at 16 38 53" src="https://user-images.githubusercontent.com/1616097/60302685-17336180-98fa-11e9-96d9-21752bfe00cb.png">

Reply:
Right, I guess I need to learn what is a target/scheme. TBH it also looks like a folder when seen from GitHub and from the filesystem, so I think my ignorance can be partly excused.

Reply:
So, each type of tests is a different target? There is a scheme for each target? Is this understanding correct? Can you also clarify the "it's not a folder" part?

Reply:
Xcode by default create a folder for every target, if you create OONI Probe Watch for example will create a folder with that name

Reply:
See different test targets here
<img width="314" alt="Screenshot 2019-06-27 at 16 52 31" src="https://user-images.githubusercontent.com/1616097/60303430-fd931980-98fb-11e9-807f-7082a93f9d10.png">

Reply:
I see. Then, we cannot easily rename now. It's fine to put there also non-unit tests. We'll rename later. :-)

Reply:
Yeah, I guess there's no immediate action for now.